diff/parse.go

```diff
@@ -612,6 +637,18 @@ func (r *HunksReader) ReadAllHunks() ([]*Hunk, error) {
 	}
 }
 
+// parseOnlyInMessage checks if line is a "Only in {source}: {filename}" and returns source and filename
+func parseOnlyInMessage(line []byte) (bool, []byte, []byte) {
+	re := regexp.MustCompile("Only in ([\x21-\x7E]+): ([\x21-\x7E]+)")
```

Comment:
Changed to non-regex check for now.

Reply:
I think the comment was about having the regexp as a package-scoped var like this:

```
var (
  reOnlyIn := regexp.MustCompile("Only in ([\x21-\x7E]+): ([\x21-\x7E]+)")
)
```

Then the regexp is compiled at load time of the library and if there is an issue it fails right away rather than in the middle of program execution.

Reply:
Yes, but in the comment below there was suggested a change to use less strict message check without regex.
Please see the updated version of [`parseOnlyInMessage`](https://github.com/sofiia-tesliuk/go-diff/blob/29fa046e33bb9ccb5992240fe27380b709e750d0/diff/parse.go#L641).